### ISSUE TITLE
Whem you dissmised the application stall

### ISSUE THREAD

**Issue by AlfredoZuloaga** [letsar/flutter_slidable#24]
I tested with your sample file and delete every row on the table doing a full swipe 
I got an alert "item will be deleted" After that the application stop responding

```shell
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.

/flutter (30767): ══╡ EXCEPTION CAUGHT BY ANIMATION LIBRARY ╞═════════════════════════════════════════════════════════
I/flutter (30767): The following assertion was thrown while notifying status listeners for AnimationController:
I/flutter (30767): Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): At this point the state of the widget's element tree is no longer stable. To safely refer to a
I/flutter (30767): widget's ancestor in its dispose() method, save a reference to the ancestor by calling
I/flutter (30767): inheritFromWidgetOfExactType() in the widget's didChangeDependencies() method.
I/flutter (30767):
I/flutter (30767): When the exception was thrown, this was the stack:
I/flutter (30767): #0      Element._debugCheckStateIsActiveForAncestorLookup.<anonymous closure> (package:flutter/src/widgets/framework.dart:3215:9)
I/flutter (30767): #1      Element._debugCheckStateIsActiveForAncestorLookup (package:flutter/src/widgets/framework.dart:3224:6)
I/flutter (30767): #2      Element.visitAncestorElements (package:flutter/src/widgets/framework.dart:3305:12)
I/flutter (30767): #3      Notification.dispatch (package:flutter/src/widgets/notification_listener.dart:59:12)
I/flutter (30767): #4      _SlidableState&State&TickerProviderStateMixin&AutomaticKeepAliveClientMixin._ensureKeepAlive (package:flutter/src/widgets/automatic_keep_alive.dart:345:49)
I/flutter (30767): #5      _SlidableState&State&TickerProviderStateMixin&AutomaticKeepAliveClientMixin.updateKeepAlive (package:flutter/src/widgets/automatic_keep_alive.dart:366:9)
I/flutter (30767): #6      SlidableState._handleShowAllActionsStatusChanged (package:flutter_slidable/src/widgets/slidable.dart:993:5)
I/flutter (30767): #7      _AnimationController&Animation&AnimationEagerListenerMixin&AnimationLocalListenersMixin&AnimationLocalStatusListenersMixin.notifyStatusListeners (package:flutter/src/animation/listener_helpers.dart:161:19)
I/flutter (30767): #8      AnimationController._checkStatusChanged (package:flutter/src/animation/animation_controller.dart:531:7)
I/flutter (30767): #9      AnimationController._startSimulation (package:flutter/src/animation/animation_controller.dart:475:5)
I/flutter (30767): #10     AnimationController.animateWith (package:flutter/src/animation/animation_controller.dart:462:12)
I/flutter (30767): #11     AnimationController.fling (package:flutter/src/animation/animation_controller.dart:450:12)
I/flutter (30767): #12     SlidableState.close (package:flutter_slidable/src/widgets/slidable.dart:903:28)
I/flutter (30767): #13     SlidableController.activeState= (package:flutter_slidable/src/widgets/slidable.dart:593:19)
I/flutter (30767): #14     SlidableState.dispose (package:flutter_slidable/src/widgets/slidable.dart:889:24)
I/flutter (30767): #15     StatefulElement.unmount (package:flutter/src/widgets/framework.dart:3833:12)
I/flutter (30767): #16     _InactiveElements._unmount (package:flutter/src/widgets/framework.dart:1697:13)
I/flutter (30767): #17     _InactiveElements._unmount.<anonymous closure> (package:flutter/src/widgets/framework.dart:1695:7)
I/flutter (30767): #18     SingleChildRenderObjectElement.visitChildren (package:flutter/src/widgets/framework.dart:4654:14)
I/flutter (30767): #19     _InactiveElements._unmount (package:flutter/src/widgets/framework.dart:1693:13)
I/flutter (30767): #20     ListIterable.forEach (dart:_internal/iterable.dart:39:13)
I/flutter (30767): #21     _InactiveElements._unmountAll (package:flutter/src/widgets/framework.dart:1706:25)
I/flutter (30767): #22     BuildOwner.finalizeTree.<anonymous closure> (package:flutter/src/widgets/framework.dart:2341:27)
I/flutter (30767): #23     BuildOwner.lockState (package:flutter/src/widgets/framework.dart:2173:15)
I/flutter (30767): #24     BuildOwner.finalizeTree (package:flutter/src/widgets/framework.dart:2340:7)
I/flutter (30767): #25     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding&WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:655:18)
I/flutter (30767): #26     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding&PaintingBinding&RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:214:5)
I/flutter (30767): #27     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:990:15)
I/flutter (30767): #28     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:930:9)
I/flutter (30767): #29     _WidgetsFlutterBinding&BindingBase&GestureBinding&ServicesBinding&SchedulerBinding._handleDrawFrame (package:flutter/src/scheduler/binding.dart:842:5)
I/flutter (30767): #30     _invoke (dart:ui/hooks.dart:128:13)
I/flutter (30767): #31     _drawFrame (dart:ui/hooks.dart:117:3)
I/flutter (30767):
I/flutter (30767): The AnimationController notifying status listeners was:
I/flutter (30767):   AnimationController#305da(◀ 1.000)
I/flutter (30767): ════════════════════════════════════════════════════════════════════════════════════════════════════
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: Looking up a deactivated widget's ancestor is unsafe.
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.
I/flutter (30767): Another exception was thrown: NoSuchMethodError: The method 'stop' was called on null.
```

**Comment by letsar** [comment]
Hi @AlfredoZuloaga 
I will look at this

**Comment by letsar** [comment]
Should be fixed in version 0.4.2
